packages/database/tsconfig.json

```
@@ -3,6 +3,11 @@
   "compilerOptions": {
     "outDir": "dist"
   },
-  "include": ["src/**/*.js"],
-  "exclude": ["src/__tests__", "src/migrations"]
+  "include": [
+    "src/**/*.js",
```

```suggestion
    "src/**/*.js"
```

I understand tsconfig is treated as a JSONC file, but, tentatively, I *think* we tend to use strict JSON anyway?